### PR TITLE
Add a note about updating and fix env variable

### LINKED ISSUE #121287: Cleanup the dev/ci/docker_linux/Dockerfile with changes requested from previous review
In order to unblock the tree changes were made to get the cirrus tests working again but not all changes were made due to time constraints. 

Initial P0: https://github.com/flutter/flutter/issues/121242
PR: https://github.com/flutter/flutter/pull/121251

### PR DESCRIPTION
**Pull request by ricardoamador**
Some changes to add to the dockerfile suggested in the initial Pull request. They were not added then in an effort to unblock the tree.

*List which issues are fixed by this PR. You must list at least one issue.*
Fixes https://github.com/flutter/flutter/issues/121287

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/ci/docker_linux/Dockerfile [hunk 1]
```diff
@@ -70,9 +70,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 ENV JAVA_HOME="/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64"
 
-RUN export CLASSPATH='/usr/share/jaxb/lib'
+ENV CLASSPATH="/usr/share/jaxb/lib"
```
Comment (christopherfujino) [inline dev/ci/docker_linux/Dockerfile:73]: do we need this line since it seems to be passing without it?

Comment (christopherfujino) [reply dev/ci/docker_linux/Dockerfile:73]: or rather, it passes when it's not affecting subsequent steps

Comment (ricardoamador) [reply dev/ci/docker_linux/Dockerfile:73]: That is a good point. Let me see what happens if it is removed.

**Review by christopherfujino — APPROVED** [review]
LGTM

**Pull request merged**